runtime/gc_glue_java/MarkingDelegate.hpp

```diff
@@ -120,10 +122,10 @@ class MM_MarkingDelegate
 	getObjectScanner(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, void *scannerSpace, MM_MarkingSchemeScanReason reason, uintptr_t *sizeToDo)
 	{
 		J9Class *clazz = J9GC_J9OBJECT_CLAZZ(objectPtr, env);
-		UDATA const referenceSize = env->compressObjectReferences() ? sizeof(uint32_t) : sizeof(uintptr_t);
+		uintptr_t const referenceSize = env->compressObjectReferences() ? sizeof(uint32_t) : sizeof(uintptr_t);
 
 		/* object class must have proper eye catcher */
-		Assert_MM_true((UDATA)0x99669966 == clazz->eyecatcher);
+		Assert_MM_true((uintptr_t)0x99669966 == clazz->eyecatcher);
```

Comment:
Can't believe we don't have J9CLASS_EYECATCHER defined somewhere in a header file to use here!? Ah, whatever.